.github/workflows/avm.template.module.yml

```diff
@@ -25,7 +25,10 @@ env:
   ARM_MGMTGROUP_ID: "${{ secrets.ARM_MGMTGROUP_ID }}"
   ARM_TENANT_ID: "${{ secrets.ARM_TENANT_ID }}"
   TOKEN_NAMEPREFIX: "${{ secrets.TOKEN_NAMEPREFIX }}"
-
+  AZURE_CLIENT_ID: "${{ secrets.AZURE_CLIENT_ID }}"
```

Comment:
Remove

Reply:
> Remove OIDC changes

This is for testing purposes only. After all changes are complete, we will revert the changes to these two files.

Reply:
Sounds fair. Thanks @jianingwang123